### PR TITLE
Bug/mc 10135 fix api call format relates to external ip

### PR DESCRIPTION
**Pull request by JamieWulala**

### CODE REVIEW

File: source/includes/gcp/_load_balancer.md [hunk 1]
```diff
@@ -120,12 +155,12 @@ Required | &nbsp;
 `shortBackend`<br/>*string* | The existing backend serivce that will be attached to this load balancer.
 `shortProtocol`<br/>*string* | The protocol of this resource.
 `shortPort`<br/>*string* | The port number of this resource.
-`shortCertificate`<br/>*string* | The name of the SSL certificate that will be attached to the target proxy if HTTPS is selected. (Required if HTTPS is selected for shortProtocol)
```
Comment (fyrnaga) [inline source/includes/gcp/_load_balancer.md]: why has it been remove. You have it in your examples

File: source/includes/gcp/_vpn_gateways.md [hunk 2]
```diff
@@ -1 +1,189 @@
...
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/vpngateways/:id</code>
+
+Retrieve a VPN gatewau in an [environment](#administration-environments).
+
+| Attributes                       | &nbsp; |
+| -------------------------------- | ------ |
+| `creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format. |
```
Comment (fyrnaga) [inline source/includes/gcp/_vpn_gateways.md:150]: missing description

File: source/includes/gcp/_vpn_gateways.md [hunk 3]
```diff
@@ -1 +1,189 @@
...
+| `region`<br/>*string*            | The self link of the region |
+| `network`<br/>*string*           | The self link of the network |
+| `status`<br/>*string*            | The status of the this resource |
+| `selfLink`<br/>*string*          | Server-defined URL for the resource. |
+| `forwardingRules`<br/>*List<Objects>*  | The forwarding rules that this resource attached to |
+| `kind`<br/>*string*              | Type of the resource. |
+| `externalIp`<br/>*Object*        | The external IP this resource is attached to |
```
Comment (fyrnaga) [inline source/includes/gcp/_vpn_gateways.md]: is there a description of of the externalip object... maybe we should have a link to it?

File: source/includes/gcp/_vpn_gateways.md [hunk 4]
```diff
@@ -1 +1,189 @@
...
+
+Retrieve a list of all VPN gateways in an [environment](#administration-environments).
+
+
+| Attributes                       | &nbsp; |
+| -------------------------------- | ------ |
+| `creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format. |
+| `region`<br/>*string*            | The self link of the region |
+| `network`<br/>*string*           | The self link of the network |
+| `status`<br/>*string*            | The status of the this resource |
```
Comment (fyrnaga) [inline source/includes/gcp/_vpn_gateways.md]: is there an enum of values?

Comment (JamieWulala) [reply source/includes/gcp/_vpn_gateways.md]: it is a enum, but we use put string here (at least anywhere else in the doc). I'll add enum values

Comment (fyrnaga) [reply source/includes/gcp/_vpn_gateways.md]: remember it is an API. It is frustrating to have to discover the possible values. Better have them documented for us and our customer

Comment (JamieWulala) [reply source/includes/gcp/_vpn_gateways.md]: fd3c96eba72d287295e385d0a0a8571ec858bd18

File: source/includes/gcp/_vpn_gateways.md (hunk 2 above)
Comment (JamieWulala) [reply source/includes/gcp/_vpn_gateways.md:150]: fd3c96eba72d287295e385d0a0a8571ec858bd18

File: source/includes/gcp/_vpn_gateways.md (hunk 3 above)
Comment (JamieWulala) [reply source/includes/gcp/_vpn_gateways.md]: fd3c96eba72d287295e385d0a0a8571ec858bd18

**Review by fyrnaga — APPROVED** [review]
LGTM

**Review by evanluc — APPROVED** [review]
Gr8!

**Pull request merged**